chapter-06/integration-tests/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
```

Comment:
Shouldn't be this folder be called `tests` and not `integration-tests`?

I believe that if we use tests, cargo will automatically compile this binary for us when running the tests...

https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html